lib/ometajs/core.js

```diff
@@ -451,7 +460,10 @@
 			};
 			this._superApplyWithArgs = function(recv, rule) {
 				var origInput = recv.input,
-					ruleArgs = arguments.length > 2 ? __slice.call(arguments, 2) : [];
+					ruleArgs = new Array(Math.min(arguments.length - 2, ruleFnArity));
```

Comment:
ruleFnArity not defined here, and wasn't used before, maybe copy paste from elsewhere?